src/main/java/config/TestProperties.java

```diff
@@ -0,0 +1,23 @@
+package config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class TestProperties {
+    private static final InputStream TEST_PROPS = TestProperties.class.getResourceAsStream("/config.properties");
+    private static final Properties PROPERTIES;
+
+    static {
+        PROPERTIES = new Properties();
+        try {
+            PROPERTIES.load(TEST_PROPS);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
if properties file was not loaded you will get NPE's further, which is hard to identify where we got this error. I recommend to define a custom exception like BadConfigException (extends RuntimeException) and throw it here